### ISSUE TITLE
Notes on existing Beginner WSG Documentation

### ISSUE THREAD

**Issue by JamesChristie-SustainableUX** [w3c/sustainableweb-ig#238]
I wanted to corral all the WSG resources (current and planned), especially the ones aimed at a less-technical audience (me). 
Including the off-site resources. In accessibility-land, for years designers told other designers to go look at wuhcag.com instead of the official documentation (sadly, WUHCAG no longer hosts a free & pretty WCAG checklist). 

I feel like I am missing some resources, please let me know. 

**Notes**

- Did there used to be a WSG overview page (similar to https://www.w3.org/WAI/standards-guidelines/wcag/ )? I swear there was but now I can’t find it. If it doesn't exist, perhaps we should make it. 
- People may be looking for a non-W3C version of the guidelines, similar to how WUHCAG simplified things for designers. Would it be acceptable to link out to such resources as they appear? 
- Off-site WSG references - it’s not clear which version of WSG they sync to. 
- WSG at a Glance isn’t mentioned in the top of the guidelines (in the abstract) - you have to find it in 1.6. Mentioning it earlier would give non-technical readers an off-ramp to something they can comprehend.  
- WSG at a Glance and Quick Reference don’t seem to reference each other. 
- At a Glance and Quick Reference sound similar. Perhaps they should be more strongly differentiated? Merged? Would renaming the QR as a checklist make sense?  

**Resource list:** 
[The guidelines](https://www.w3.org/TR/web-sustainability-guidelines/ ):  
(although more sites link to https://w3c.github.io/sustyweb/ ) 

[Abstract](https://www.w3.org/TR/web-sustainability-guidelines/#abstract ): 
As the first thing visitors might read, I wonder if there should be a direct link to the At A Glance. 

[WSG at a glance ](https://w3c.github.io/sustainableweb-wsg/glance.html )
A concise overview of WSG and its contents, featuring highlights selected by members of the Interest Group. 

[WSG quick reference](https://w3c.github.io/sustainableweb-wsg/quickref.html)
A quick-reference guide to WSG, including a print-friendly checklist for easy navigation through the guidelines. 

**Offsite**
`[https://sustainablewebdesign.org/](https://sustainablewebdesign.org/)`
Note: no version number, when was it updated? 
Owners: Mightybytes + Wholegrain

[https://www.mightybytes.com/insights/web-sustainability-guidelines/](https://www.mightybytes.com/insights/web-sustainability-guidelines/)
Owner: Mightybytes

**Comment by AlexDawsonUK** [comment]
Regarding your points:

1. The overview page is largely explored by our two repo readme files ([WSG](https://w3c.github.io/sustainableweb-wsg/README.html) and [IG](https://w3c.github.io/sustainableweb-ig/)). Note that in the WSG repo readme, in the work section there is a complete list of our deliverables.
2. Absolutely, [I've been maintaining a campaign toolkit](https://github.com/w3c/sustainableweb-ig/blob/main/campaign-toolkit.md) that references off-site implementations, research, articles, talks, et al, that refers to our work. I've mentioned it in the Slack and meetings numerous times, and it's particularly useful for showing the growth of adoption of our work (good for conference talks!).
3. RE Off-site WSG references: We don't control links outside W3C, so as a policy we can't enforce people state what version of the spec they are using. That being said, the default should always be the most recent one as we are still in-draft.
4. At-A-Glance is mentioned at the top of the spec, in the "More details about this document" bit of the heading (Supplements), which lists all of our deliverables (a common pattern with all W3C specs).
5. ~~Noted that at-a-glance and quick ref could reference each other, I'll make a note for that.~~ Done. ☑️
6. Certainly an idea, we were talking about the fact that at-a-glance on it's own may not be enough to warrant a NOTE so merging with the quick reference could make sense.

**Comment by AlexDawsonUK** [comment]
Additional note that we do have [the explainer](https://github.com/w3c/sustainableweb-ig/blob/main/explainer.md) that may also cover this.
